src/mds/ScrubStack.cc

```diff
@@ -663,7 +663,11 @@ void ScrubStack::scrub_status(Formatter *f) {
     else
       f->dump_stream("path") << "#" << header->get_origin();
 
-    f->dump_string("tag", header->get_tag());
+    if (mdcache->get_inode(header->get_origin())->is_mdsdir() && tag.empty()) {
+      f->dump_string("tag", "scrub_mdsdir_at_root");
```

Comment:
This is going to confuse users _a lot_. If there any need to show that mdsdir is included in scrub. Maybe introducing a `flags` section in the json and including it there?

Reply:
Sounds good, will try that out

Reply:
this change has been incorporated. thanks

Reply:
we already have `options`, made changes in https://github.com/ceph/ceph/pull/47649/commits/3badfe7171cad89e25255f805ec241c3d12166f3 to print `scrub_mdsdir`